lib/storage/file.js

```diff
@@ -932,22 +932,29 @@ File.prototype.getMetadata = function(callback) {
  * @param {object} callback.policy - The document policy.
  *
  * @example
- * file.getSignedPolicy({
+ * var TWO_WEEKS_FROM_NOW = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
```

Comment:
Will this work with something like `moment.duraction(2, 'weeks');` ? (http://momentjs.com/docs/#/durations/creating/)